Cargo.toml

```diff
@@ -13,7 +13,7 @@ publish = false
 autobenches = false
 
 [dependencies]
-aurora-engine-types = { git = "https://github.com/aurora-is-near/aurora-engine.git", branch = "develop", default-features = false, features = ["borsh-compat"] }
+aurora-engine-types = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "3.5.0", default-features = false, features = ["borsh-compat"] }
```

Comment:
Probably we also need to remove the 'borsh-compt', let me check

Reply:
Yeah, the `borsh-compt` enable the `borsh 0.9`, so it should be removed to be able to compile aurora eth connector tests.